odoo/addons/base/migrations/13.0.1.3/openupgrade_analysis.txt

```diff
@@ -303,6 +302,21 @@ NEW res.country.state: base.state_cn_XJ
 NEW res.country.state: base.state_cn_XZ
 NEW res.country.state: base.state_cn_YN
 NEW res.country.state: base.state_cn_ZJ
+NEW res.country.state: base.state_ee_37
```

Comment:
Are you sure these are new states?

Reply:
hey, I used last openupgrade v12 and v13 for real. Thus, what do you want me to do in these comments o yours?

Reply:
OK, just confirm that you are using in both parts the latest version, for not adding changes that are not real (for example, these states may be added both in v12 and v13).

Reply:
https://github.com/odoo/odoo/commit/882fa61fd3c3e0e332fb62ecb92a6c4416864321